scripts/build.sh

```diff
@@ -96,8 +98,11 @@ curl -sSL https://github.com/contiv/netplugin/releases/download/$contiv_version/
 pushd $output_dir
 tar oxf netplugin-$contiv_version.tar.bz2 netctl
 rm -f netplugin-$contiv_version.tar.bz2
-git clone https://github.com/contiv/ansible
 popd
+# add ansible repo contents where final tarball will include
+mkdir $output_dir/ansible
```

Comment:
Can we delete this directory before recreating it?  Also, can we add this to `.gitignore`?

Reply:
L80 already deleted this directory

Reply:
"release" is in .gitignore: https://github.com/contiv/install/blob/master/.gitignore#L7
and release is parent of outputdir:
https://github.com/contiv/install/blob/master/scripts/build.sh#L71

Reply:
👍